benchmarks/vectorized_channel_shuffle.py

```diff
@@ -48,7 +48,7 @@ class OldChannelShuffle(BaseImageAugmentationLayer):
             ` or (width, height, channels)`, with dtype
                 tf.float32 / tf.uint8
         training: A boolean argument that determines whether the call should be
-            run in inference mode or training mode, defaults to True.
+            run in inference mode or training mode, defaults to `True`.
```

Comment:
Let's standardize on always making this "defaults to xyz" it's own sentence at the end of the docstring (instead of using a comma to just have this as an extra clause at the end of a sentence)

Reply:
Ok, done